python/lsst/summit/extras/logUtils.py

```diff
@@ -64,13 +77,23 @@ class LogBrowser:
         "with gufunc signature (n?,k),(k,m?)->(n?,m?)",
     ]
 
-    def __init__(self, butler, taskName, collection):
+    def __init__(self, butler, taskName, collection, whereStr="", bindMap=None):
         self.taskName = taskName
         self.collection = collection
+        self.whereStr = whereStr
+        self.bindMap = bindMap
 
         self.log = _LOG.getChild("logBrowser")
         self.butler = butler
 
+        if self.bindMap is not None:
+            for key in self.bindMap.keys():
+                if key not in self.whereStr:
+                    self.log.warn(
+                        f"Key '{key}' in bindMap is not in the whereStr provided:\n"
+                        f"{self.whereStr}\nso no binding will take effect."
```

Comment:
Do you really want two newlines inside the single log message? That's unusual. I'm not even sure one is recommended tbh, though it's `summit_extras`, so I don't care much.